tools/rna_tools/trna_prediction/aragorn_out_to_gff3.py

```diff
@@ -10,6 +10,31 @@
 KEY_ORDER = ('parent', 'source', 'type', 'start', 'end', 'score', 'strand',
              '8', 'quals')
 
+# Table of amino acids
+aa_table = {
+    'Ala' : 'A',
+    'Arg' : 'R',
+    'Asn' : 'N',
+    'Asp' : 'D',
+    'Cys' : 'C',
+    'Gln' : 'Q',
+    'Glu' : 'E',
+    'Gly' : 'G',
+    'His' : 'H',
+    'Ile' : 'I',
+    'Leu' : 'L',
+    'Lys' : 'K',
+    'Met' : 'M',
+    'Phe' : 'F',
+    'Pro' : 'P',
+    'Ser' : 'S',
+    'Thr' : 'T',
+    'Trp' : 'W',
+    'Tyr' : 'Y',
+    'Val' : 'V',
+    'Pyl' : 'O',
+    'seC' : 'U',
```

Comment:
No. Aragorn abbreviates selenocysteine as `seC`, though that seems to be unique to Aragorn. Wikipedia uses `Sec`.
Example Aragorn output:
`5   tRNA-seC               [244644,244785]	87  	(tca)i(29,57)`